packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -1110,4 +1126,29 @@ export class InputDatePicker
           )
           .join("")
       : "";
+
+  private normalizeToCurrentCentury(twoDigitYear: number) {
```

Comment:
Would this be beneficial as a util outside of the component that other components may use? Or is this the only place a user would type in a year?

Reply:
One component i couldn't think of is date-picker but there is a strange behavior in date-picker which is normalizing year to 19th century when user types any two digit year.